handlers/routes.py

```diff
@@ -1009,55 +1013,55 @@ def render_contract(user, tmpdir, nda_file_base64, contract_file_name,  signer_u
                          render_nda_only, signer_user, link_id, doc_file_name="", contract_file_name="",
                          email_body_html="", email_body_text=""):
     """Renders the documents and if needed send it to cryptosign and finally send it by email"""
 
     attachments_list = []
-    doc_id = error = ""
+    doc_id = error = errornda = errorwp = ""
     mymail = Mailer(username=conf.SMTP_USER, password=conf.SMTP_PASS, host=conf.SMTP_ADDRESS, port=conf.SMTP_PORT)
 
     # Here we create a temporary directory to store the files while the function sends it by email
     with tempfile.TemporaryDirectory() as tmp_dir:
         try:
 
             if render_nda_only is False:
-                attachments_list, error = render_document(tmp_dir, thisdoc, doc_file_name, user, google_credentials_info,
+                attachments_list, errornda = render_document(tmp_dir, thisdoc, doc_file_name, user, google_credentials_info,
                                                           signer_user, attachments_list)
             if render_wp_only is False:
-                attachments_list, error = render_contract(user, tmp_dir, nda_file_base64,
+                attachments_list, errorwp = render_contract(user, tmp_dir, nda_file_base64,
                                                           contract_file_name, signer_user, attachments_list, link_id)
-
+            error = errornda + errorwp
             if error != "":
-                return render_template('pdf_form.html', id=doc_id, error=error)
-            if not email_body_html:
-                email_body_html = DEFAULT_HTML_TEXT
-
-            # send the email with the result attachments
-            sender_format = "{} <{}>"
-            loader = Loader("templates/email")
-            button = loader.load("cta_button.html")
-            notification_subject = "Your Document {} has been downloaded".format(thisdoc.doc_id)
-            analytics_link = "{}{}analytics/{}".format(conf.BASE_URL, BASE_PATH, thisdoc.doc_id)
-
-            mymail.send(subject=thisdoc.wp_name, email_from=sender_format.format(user.org_name, conf.SMTP_EMAIL),
-                        emails_to=[signer_user.email],
-                        attachments_list=attachments_list,
-                        html_message=email_body_html + button.generate().decode("utf-8"),
-                        text_message=email_body_text)
-
-            html_text = NOTIFICATION_HTML.format(signer_user.email, thisdoc.doc_id, analytics_link, analytics_link)
-            mymail.send(subject=notification_subject,
-                        attachments_list=attachments_list,
-                        email_from=sender_format.format("WPCI Admin", conf.SMTP_EMAIL),
-                        emails_to=[user.org_email], html_message=html_text,
-                        text_message=email_body_text)
+                logger.info("error not empty {}".format(error))
```

Comment:
Puedes aclarar este logger?  Me refiero al texto `error not empty` no dice mucho, dime que trataste de decir y podemos checarlo

Reply:
ah ok, lo que sucede es que cuando intenta renderizar un document o un nda (cada uno tiene su metodo distinto porque se realizan acciones distintas) retorna el error y esos errores se concatenan entonces el error not empty quiere decir que al menos uno de estos dos metodos dio error (no siempre se utilizan los dos juntos) y pues el log es mas que todo para imprimir esos errores que ocurrieron alli

Reply:
voy a cambiar ese texto por (there was an error on the documents rendering)